### PR TITLE
fixed issue #211 jvm args are appended two times

### PR DESCRIPTION
**Pull request by missedone**
fixed issue #211

### CODE REVIEW

File: testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java [hunk 1]
```diff
@@ -355,18 +355,6 @@ protected VMRunnerConfiguration createVMRunner(
   }
 
   @Override
-  public String getVMArguments(ILaunchConfiguration configuration)
-      throws CoreException {
-    StringBuilder vmArgs = new StringBuilder();
-    vmArgs.append(super.getVMArguments(configuration));
```
Comment (missedone) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java]: the 1st call retrieves `org.eclipse.jdt.launching.VM_ARGUMENTS` and append to testng JVM process

File: testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java [hunk 2]
```diff
@@ -355,18 +355,6 @@ protected VMRunnerConfiguration createVMRunner(
   }
 
   @Override
-  public String getVMArguments(ILaunchConfiguration configuration)
-      throws CoreException {
-    StringBuilder vmArgs = new StringBuilder();
-    vmArgs.append(super.getVMArguments(configuration));
-
-    vmArgs.append(" ");
-    vmArgs.append(ConfigurationHelper.getJvmArgs(configuration));
```
Comment (missedone) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java]: the 2nd call retrieves `org.eclipse.jdt.launching.VM_ARGUMENTS` and append to testng JVM process
so, the fix is simply call ConfigurationHelper.getJvmArgs(configuration) to make sure only have one `org.eclipse.jdt.launching.VM_ARGUMENTS` append to testng

Comment (cbeust) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java]: I just deployed a new plug-in with this fix.

-- 
Cédric

Comment (missedone) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java]: hi @cbeust 
looks like the testng.jar in the new plugin was built with java 8:
`java.lang.UnsupportedClassVersionError: org/testng/remote/RemoteTestNG : Unsupported major.minor version 52.0`

Comment (cbeust) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java]: Just pushed a fix, should be version 51, can you confirm?

-- 
Cédric

Comment (cbeust) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java]: What I see on my end:

$ javap -v -classpath testng-eclipse-plugin/lib/testng.jar
org.testng.TestNG|grep ajor
  major version: 51

​

-- 
Cédric

Comment (missedone) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java]: yep, i think i was pointing to the wrong testng version in pom.xml which was built with java 8.

anyway, looks like the plugin not including the patch:
```
============== VM args =============
-Dx=y
-ea
-Dx=y
```

could you have a check?

Comment (cbeust) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java]: Nothing came down when I pulled again, but I deployed a new one anyway. I
confirm your change and its merge are in my log:

commit 08d55a9f734272babc1ee89f6ca7fbc641570fac
Merge: 0d4fccf 12a87f6
Author: Cedric Beust <cedric@beust.com>
Date:   Wed Dec 23 22:00:14 2015 +0400

    Merge pull request #212 from missedone/issue/211

    fixed issue #211 jvm args are appended two times

commit 12a87f64dd9a9a2b5a27c84d959ac75eec1aa85b
Author: Nick Tan <missedone@gmail.com>
Date:   Wed Dec 23 06:30:18 2015 -0800

    fixed issue #211 jvm args are appended two times

Is the new one ok?
​

Comment (missedone) [inline testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java]: this one works now, thanks

**Pull request merged**